FWCore/Services/plugins/InitRootHandlers.cc

```diff
@@ -875,6 +875,10 @@ namespace edm {
       if(debugLevel >0) {
 	gDebug = debugLevel;
       }
+
+      // Enable Root implicit multi-threading
+      bool imt = pset.getUntrackedParameter<bool>("EnableIMT");
+      if (imt) ROOT::EnableImplicitMT(4);
```

Comment:
Is "4" is a number we don't expect to change? it looks like it should be configurable (or even correlated with the # of threads used in the job)